src/Core/Managed/Shared/Channel/InMemoryTransmitter.cs

```diff
@@ -94,7 +94,7 @@ private void Runner()
                 while (this.enabled)
                 {
                     // Pulling all items from the buffer and sending as one transmissiton.
-                    this.DequeueAndSend();
+                    this.DequeueAndSend(timeout: default(TimeSpan)); // use default timeout
```

Comment:
It is actually using the default timeout which is not zero. If we pass default, it uses some predefined value on Transmission level. I can make comment more clear.